main.py

```diff
@@ -1,12 +1,243 @@
+from flask import json
 import requests
+from requests.api import request
 
 URL = "http://127.0.0.1:5000"
 BACKUP_URL = "https://retro-video-store-api.herokuapp.com"
 
-def main():
-    print("WELCOME TO RETRO VIDEO STORE")
+
+'''
+    =============================================
+    HELPER PRINTS
+    =============================================
+'''
+
+def bar_break():
+    print("\n==========================\n")
+
+def list_options_ee():
+    options = {
+        "1" : "Add Video to Store Stock",
+        "2" : "Edit Video Info",
+        "3" : "Remove Video From Inventory",
+        "4" : "View Current Store Stock",
+        "5" : "View Video Info",
+        "6" : "Add New Customer",
+        "7" : "Edit Existing Customer",
+        "8" : "Delete Existing Customer",
+        "9" : "View Existing Customer Records",
+        "10" : "View All Existing Customers",
+        "11" : "Check Out",
+        "12" : "Check In"
+    }
+
+    bar_break()
+    print("Here are your available options:\n")
+    for choice in options:
+        print(f"Option {choice}. {options[choice]}")
+
+    bar_break()
+
+    return options
+
+def list_options_cust():
+    options = {
+
+    }
     pass
 
+'''
+    =============================================
+    EMPLOYEE OPTION FUNCTIONS
+    =============================================
+'''
+
+def add_video():
+    print("Enter video info below:")
+    request_body = {}
+    request_body["title"] = input("Title: ")
+    request_body["release_date"] = input("Release date: ")
+    request_body["total_inventory"] = input("Total inventory: ")
+
+    response = requests.post(URL +"/videos", json=request_body)
+    print(json.dumps(response.json(), indent=1))
+    return
```

Comment:
An explicit return is not required at the end of a function if there is no return value.  At the end of a function, Python will return a default of None if there is no return value.